sdk/python/kfp/dsl/type_utils.py

```diff
@@ -91,7 +91,7 @@ def get_artifact_type_schema(
     artifact_class = artifact_class_or_type_name
 
   return pipeline_spec_pb2.ArtifactTypeSchema(
-      schema_title=artifact_class.TYPE_NAME)
+      schema_title=artifact_class.TYPE_NAME, schema_version="0.0.1")
```

Comment:
Maybe we can add `VERSION` field to artifacts, and let each artifact separately manage their schema versions. Then this line can be 
```py
return pipeline_spec_pb2.ArtifactTypeSchema(
      schema_title=artifact_class.TYPE_NAME, schema_version=artifact_class.VERSION)
```

WDYT?